### PR TITLE
[2] Checkout command name must be sd-checkout-code

### PR DESCRIPTION
**Pull request by tkyi**
Use new validation for output of `getCheckoutCommand` method

Blocked by https://github.com/screwdriver-cd/data-schema/pull/94
Related to https://github.com/screwdriver-cd/models/pull/132

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-scm-base",
-  "version": "2.4.0",
+  "version": "2.5.0",
```
Comment (FenrirUnbound) [inline package.json]: Is it backwards compatible?

Comment (d2lam) [inline package.json]: Yeah, technically. We haven't integrated this.

Comment (FenrirUnbound) [inline package.json]: Well, if it's not integrated, why bother with a minor version bump?

Comment (tkyi) [inline package.json]: I know we did that in the past to make sure we were pulling in the right version in scm-github and scm-bitbucket. But we could probably pin to a patch version

Comment (tkyi) [inline package.json]: Removing minor version bump since only the scm plugins are using this right now

**Pull request merged**